db/migrate/20161128201646_create_quote_likes.rb

```diff
@@ -0,0 +1,10 @@
+class CreateQuoteLikes < ActiveRecord::Migration
+  def change
+    create_table :quote_likes do |t|
+      t.references :user, index: true, foreign_key: true, null: false
+      t.references :quote, index: true, foreign_key: true, null: false
+
+      t.timestamps null: false
+    end
```

Comment:
This what you talking about (just change to quotes instead of reviews)
```
    execute <<-SQL.squish
      INSERT INTO review_likes (review_id, user_id, created_at, updated_at)
      SELECT target_id, user_id, created_at, updated_at
      FROM votes
      WHERE target_type = 'Review'
      AND positive = 't'
    SQL
```